### ISSUE TITLE
QAQC issues

### ISSUE THREAD

**Issue by greymonroe** [International-Soil-Radiocarbon-Database/ISRaD#1]
I am playing around with the template and now also the database - thanks again for sharing! I downloaded the dataset named soilcarbon_database.rda on Github (in soilcarbon/data). Is this the latest version of the database? 

I am asking because I got confused a bit after I extracted data from Corey's Cowlitz River chronosequence study (Lawrence et al. 2015) from the database to calculate the stocks but I could not reproduce the C stocks (mass per area) reported in that paper graphically. After that, I noticed that the C and N concentrations (c_tot, n_tot) differ from what is reported in the supplementary material to Corey´s paper (although bulk densities are the same, see full comparison in attachment). When I used data from the supplementary material instead, I could calculate the "right" C stocks showed in the graphs.

The curator of the study according to the database is Yujie He, do you think I should first get in touch with him or with Corey? I wanted to make sure that I am looking at the right data before that.

**Comment by ShaneStoner** [comment]
Hey all,
I recently encountered an error where Excel accepted variations in capitalization while R did not, which made the task of finding which rows were failing QAQC difficult/impossible. Some of the operations in the QAQC code identify which rows are failing QAQC, while others (like mismatched site/profile/fraction names) just report that there is a mismatch but does not tell the user the row in which the error occurs. I added this functionality in my local version of the QAQC and found the problematic rows in my excel file. Is there a downside to adding this to the master QAQC function? If no, I'd be happy to write/add it and make a pull request. Gotta get my name on that contributor page! Thanks for your input.
